save_awq.py

```diff
@@ -0,0 +1,101 @@
+"""Converts and store AWQ-quantized model."""
+
+import argparse
+import json
+import logging
+
+import awq
+import transformers
+
+QUANT_CONFIG = {
+    "zero_point": True,
+    "q_group_size": 128,
+    "w_bit": 4,
+    "version": "GEMM",
+}
+
+
+def parse_args():
+    """Parses command line arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--dataset",
+        type=str,
+        default="msp_open_ai_ada2_random_s5000_gpt4_da0_mr20_sampled_mix.jsonl",
```

Comment:
is it possible to have rank zephy's training data or a subset of it as the default value  of the calibration dataset?

Reply:
This would probably requires changes to the load dataset logic too

Reply:
This is the file that @ronakice shared. Wasn't this one used for training?

Reply:
No, this is not the data that we used for finetuning rankzephyr, but I leave it to Ronak to decide if we want to you the training dataset or the one that we shared with you.

Reply:
@ronakice PTAL!

Reply:
> /u3/rpradeep/RankVicuna/data/msp_open_ai_ada2_random_s5000_gpt4_da0_mr20_sampled_mix.jsonl

This is the file I used to train RankZephyr @sahel-sh?

Reply:
either way, something is off in AWQ quantizing, i will advice against merging until this is properly sorted